assets/helpers/internationalisation/country.js

```diff
@@ -91,6 +91,13 @@ function fromString(s: string): ?IsoCountry {
   }
 }
 
+export function toGuCountryCode(isoCountry: IsoCountry): string {
```

Comment:
This concept is modelled in support-internationalisation as a `CountryGroup` with possible values ca, us,  uk, au, nz, eu and int.  Maybe `toCountryGroup` might be a better name?